sdcm/ec2_client.py

```diff
@@ -1,11 +1,11 @@
-import logging
+import base64
```

Comment:
I'm guessing @KnifeyMoloko has some automatic import ordering (part of him IDE)

@KnifeyMoloko in general we would prefer not doing those, it's can complicates backports, 

for example we won't backport this one, but a different PR would add import around one of those imports
and it would cause a merge conflict for cherry-picking that commit.

if you still think this ordering is a **must**, make sure it's part of the pre-commit, and then we'll fix all the code once, and keeping it like that. same as we do for autopep8. (we just did those changes, it was painful for a while, and then we can forget about it)

Reply:
Yeah, I usually run the import optimization myself, this time I think I checked the automatic import optimization in PyCharm. Either way, since I'm not making changes in the code, I removed these changes to importing as well.